### PR TITLE
Release v0.69.0

### PR DESCRIPTION
**Pull request by hughesjj**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,13 @@
 # Changelog
 
 ## Unreleased
+## v0.69.0
+
+Updates smartagent, javaagent, and opentelemetry to `v5.27.2`, `1.20.0`, and `v0.69.0`/`v0.69.1` respectively.
```
Comment (rmfitzpatrick) [inline CHANGELOG.md:6]: mind using the existing boilerplate for otelcol? The other items could be individual statements like we've done in the past as a clear convention.

File: Makefile [hunk 2]
```diff
@@ -1,4 +1,5 @@
 include ./Makefile.Common
+SHELL := /bin/bash
```
Comment (rmfitzpatrick) [inline Makefile:2]: `Makefile.Common` already sets this if not set to respect the user's env.

File: CHANGELOG.md (hunk 1 above)
Comment (jeffreyc-splunk) [inline CHANGELOG.md:6]: For consistency:
- The otel note should match previous releases
- Move the smart agent and java agent notes to a "Enhancements" section
- Include links to all respective github releases for each version

**Review by atoulme — APPROVED** [review]

File: go.mod [hunk 3]
```diff
@@ -77,7 +77,7 @@ require (
 	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/windowsperfcountersreceiver v0.69.0
 	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/zipkinreceiver v0.69.0
 	github.com/signalfx/golib/v3 v3.3.47
-	github.com/signalfx/signalfx-agent v1.0.1-0.20230104182534-9eee411fe305
+	github.com/signalfx/signalfx-agent v1.0.1-0.20230117211433-9858481024e4
```
Comment (rmfitzpatrick) [inline go.mod:80]: The agent and apm replace directives also need to be updated since the signalfx exporter shares the dep.

Comment (jeffreyc-splunk) [reply go.mod:80]: https://github.com/signalfx/splunk-otel-collector/blob/main/go.mod#L497-L498

**Comment by jeffreyc-splunk** [comment]
Should also run `make for-all CMD="make tidy"` to tidy up the smart agent extension and receiver updates.

**Review by rmfitzpatrick — APPROVED** [review]

**Pull request merged**